### PR TITLE
Task 22

### PR DESCRIPTION
**Pull request by SergeyIgnatovich9**

### CODE REVIEW

File: src/test/java/BaseTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public class BaseTest {
+    private static final String BASE_URL = TestConfig.get("url.home");
+    protected HomePage homePage;
+
+    @BeforeMethod
+    public void init() {
+        DriverManager.getDriver().get(BASE_URL);
+        homePage = new HomePage();
+    }
```
Comment (LikBez123) [inline src/test/java/BaseTest.java:15]: separate methods with empty line

File: src/test/java/TextEntryTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    @Test
+    public void frameTest() {
+        driver.switchTo().frame(FRAME_ID);
+        WebElement contentBody = driver.findElement(CONTENT_BODY);
+        contentBody.clear();
+        contentBody.sendKeys(CUSTOM_TEXT);
+        Assert.assertEquals(CUSTOM_TEXT, contentBody.getText());
+
```
Comment (LikBez123) [inline src/test/java/TextEntryTest.java]: remove extra line

**Review by LikBez123 — APPROVED** [review]

File: src/test/java/BaseTest.java (hunk 1 above)
Comment (SergeyIgnatovich9) [reply src/test/java/BaseTest.java:15]: done

File: src/test/java/TextEntryTest.java (hunk 2 above)
Comment (SergeyIgnatovich9) [reply src/test/java/TextEntryTest.java]: done

**Pull request merged**